internal/bundler/bundler.go

```diff
@@ -193,39 +181,48 @@ func parseFile(args parseArgs) {
 		},
 	}
 
+	translationsFinder := func(componentPath string) (string, bool) {
+		return getFileTranslationsDir(componentPath, &args.caches.TranslationsCache)
+	}
+
 	switch loader {
 	case config.LoaderJS:
-		source.Contents = translationsHack(args.options.SpinxAssetBaseURL, source.PrettyPath, source.Contents, &args.caches.TranslationsCache)
```

Comment:
You'd think that avoiding all of this string manipulation + fs lookups on *every* build would make a big difference, eh? It's *maybe* ~100ms faster for full builds, and ~50ms faster for focused builds.

That was testing on a Macbook.  Trying this out in Spin workspaces, it seemed ~50-100ms slower, but there's a lot of variation there.